tasks/todo.md

```
@@ -0,0 +1,131 @@
+# Plan — Peer-Sponsored Account Creation (à la hive.io /join)
+
+## Goal
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)